packages/react-cookie/README.md

```diff
@@ -0,0 +1,85 @@
+# `@shopify/react-cookie`
```

Comment:
Just roughed this out for completeness, but will put some more ❤️  into this `Readme` either before or immediately after merging.